pyClient/zethContracts.py

```diff
@@ -11,21 +11,34 @@
 
 w3 = Web3(HTTPProvider("http://localhost:8545"))
 
-def compile_contracts():
+def compile_pghr13_contracts():
     contracts_dir = os.environ['ZETH_CONTRACTS_DIR']
-    path_to_pairing = os.path.join(contracts_dir, "Pairing.sol")
-    path_to_bytes = os.path.join(contracts_dir, "Bytes.sol")
-    path_to_verifier = os.path.join(contracts_dir, "Verifier.sol")
-    path_to_mixer = os.path.join(contracts_dir, "Mixer.sol")
-    compiled_sol = compile_files([path_to_pairing, path_to_bytes, path_to_verifier, path_to_mixer])
+    path_to_verifier = os.path.join(contracts_dir, "Pghr13Verifier.sol")
+    path_to_mixer = os.path.join(contracts_dir, "Pghr13Mixer.sol")
+    compiled_sol = compile_files([path_to_verifier, path_to_mixer])
+    verifier_interface = compiled_sol[path_to_verifier + ':Verifier']
+    mixer_interface = compiled_sol[path_to_mixer + ':Pghr13Mixer']
+    return(verifier_interface, mixer_interface)
+
+def compile_groth16_contracts():
+    contracts_dir = os.environ['ZETH_CONTRACTS_DIR']
+    path_to_verifier = os.path.join(contracts_dir, "Groth16Verifier.sol")
+    path_to_mixer = os.path.join(contracts_dir, "Groth16Mixer.sol")
+    compiled_sol = compile_files([path_to_verifier, path_to_mixer])
     verifier_interface = compiled_sol[path_to_verifier + ':Verifier']
-    mixer_interface = compiled_sol[path_to_mixer + ':Mixer']
```

Comment:
Do you have an implementation of the Groth16 verifier (that goes with the "Groth16Mixer")?

Reply:
I have just removed all references of Groth16 implementation